tests/test_create_pod_attaching_multiple_pvc_start_IO.py

```diff
@@ -0,0 +1,376 @@
+import os
+import yaml
+import logging
+import base64
+import pytest
+import ocs.ocp
+import ocs.pod
+import ocs.defaults as defaults
+
+from ocs import exceptions
+from time import sleep
+from ocsci import tier1, ManageTest
+from ocsci.enums import StatusOfTest
+from utility import templating
+
+log = logging.getLogger(__name__)
+
+CBP = ocs.ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+SEC = ocs.ocp.OCP(
+    kind='Secret', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+SC = ocs.ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+PVC = ocs.ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POD = ocs.ocp.OCP(
+    kind='Pod', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+OCP = ocs.ocp.OCP(
+    kind='Service', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+DC = ocs.ocp.OCP(
+    kind='DeploymentConfig', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+TEMP_YAML = os.path.join("templates/ocs-deployment", "temp.yaml")
+TEMPLATES_DIR = "templates/CSI/rbd"
+CIRROS_DIR = "templates/app-pod-yamls"
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Removing the resources created for this test
+    """
+    self = request.node.cls
```

Comment:
no need for this as you are not using any class attributes in setup() or teardown()

Reply:
ohh ok. Will remove this line.